grr/server/grr_response_server/output_plugins/email_plugin.py

```diff
@@ -89,10 +89,15 @@ def ProcessResponse(self, response):
         signature=config.CONFIG["Email.signature"],
         hostname=utils.SmartUnicode(hostname),
         creator=utils.SmartUnicode(self.token.username))
-
+    
+    if config.CONFIG["Worker.smtp_mailuser"]:
```

Comment:
Instead of this `if`-statement you should specify a default value when you define config option. It also helps to avoid code duplication with the other file.